### PR TITLE
Add `alpha` argument to `adapted_rand_error` 

### PR DESCRIPTION
**Pull request by lucas-johnson**
Resolves https://github.com/scikit-image/scikit-image/issues/6433

### CODE REVIEW

File: skimage/metrics/_adapted_rand_error.py [hunk 1]
```diff
@@ -20,12 +20,24 @@ def adapted_rand_error(image_true=None, image_test=None, *, table=None,
...
-        The adapted Rand error; equal to :math:`1 - \frac{2pr}{p + r}`,
-        where ``p`` and ``r`` are the precision and recall described below.
+        The adapted Rand error; equal to
+        :math:`1 - \frac{\sum_{ij} p_{ij}^{2}}{\alpha \sum_{k} s_{k}^{2} +
+        (1-\alpha)\sum_{k} t_{k}^{2}}`,
+        where :math:`p_{ij}` is the probability that a pixel has the same label
+        in the test image *and* in the true image, :math:`t_{k}` is the
+        probability that a pixel has label :math:`k` in the true image,
+        and :math:`s_{k}` is the probability that a pixel has label :math:`k`
+        in the test image.
```
Comment (alexdesiqueira) [inline skimage/metrics/_adapted_rand_error.py:40]:
```suggestion
The adapted Rand error.
```

The remainder should be in `Notes`.

**Review by alexdesiqueira — CHANGES_REQUESTED** [review]
Thank you @lucas-johnson!
I'll ask a few changes on the documentation, but otherwise this one looks good to me.

**Comment by lucas-johnson** [comment]
Documentation tweaks made!

**Comment by lucas-johnson** [comment]
As near as I can tell that test failure looks unrelated to this PR... local pytests are passing, with the exception of benchmark tests.

**Review by alexdesiqueira — APPROVED** [review]

**Comment by alexdesiqueira** [comment]
Thank you @lucas-johnson! Looks good, thanks for this!

**Comment by alexdesiqueira** [comment]
On the CI tests, I opened an issue: #6479.

**Comment by alexdesiqueira** [comment]
@scikit-image/core please check and merge (if OK) / discuss (if not) this one. Thanks y'all!

File: skimage/metrics/_adapted_rand_error.py [hunk 2]
```diff
@@ -20,12 +20,14 @@ def adapted_rand_error(image_true=None, image_test=None, *, table=None,
     ignore_labels : sequence of int, optional
         Labels to ignore. Any part of the true image labeled with any of these
         values will not be counted in the score.
+    alpha : float, optional
+       Relative weight given to precision and recall in the adapted Rand error
+       calculation.
```
Comment (rfezzani) [inline skimage/metrics/_adapted_rand_error.py:25]:
```suggestion
        Relative weight given to precision and recall in the adapted Rand error
        calculation.
```
just aligning the tabulation ;)

**Review by rfezzani — APPROVED** [review]
Thank you @lucas-johnson for your contribution, I just found a minor formatting issue :wink:

**Review by jni — APPROVED** [review]

**Comment by rfezzani** [comment]
Pushing the green button :tada:  :slightly_smiling_face: Thank you @lucas-johnson

**Pull request merged**